src/test/java/ru/training/at/hw2/ex1/GeneralItemPlacementTest.java

```diff
@@ -0,0 +1,128 @@
+package ru.training.at.hw2.ex1;
+
+import java.util.List;
+import java.util.stream.IntStream;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+import ru.training.at.hw2.BaseJdiTestingIndexPageTest;
+import ru.training.at.hw2.data.JdiTestingIndexPageData;
+
+public class GeneralItemPlacementTest extends BaseJdiTestingIndexPageTest {
+
+    private static final String INNER_HTML_ATTRIBUTE = "innerHTML";
+
+    @Test
+    public void firstExerciseItemPlacementTest() {
+        openTestSiteByURL();
+        assertBrowserTitle();
+        performLogin();
+        assertUserName();
+        assertHeaderNavbarItems();
+        assertThereAreFourImagesOnThePage();
+        checkIfTextsUnderTheImagesAreCorrect();
+        checkIfThereIsAFrameWithButton();
+        checkFrameInnerButton();
+        switchToOriginalWindowBack();
+        assertSidebarItems();
+        assertAccumulator.assertAll();
+    }
+
+    private void assertHeaderNavbarItems() {
+        // 5. Assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> headerMenuItems = driver.findElements(Locators.HEADER_NAVBAR_ITEMS);
+
+        assertAccumulator.assertEquals(headerMenuItems.size(), JdiTestingIndexPageData.HEADER_MENU_ITEMS_TEXT.size());
+
+        IntStream.range(0, headerMenuItems.size())
+                 .peek(index -> assertAccumulator.assertTrue(headerMenuItems.get(index).isDisplayed()))
+                 .forEachOrdered(index -> {
+                     assertAccumulator.assertEquals(
+                         headerMenuItems.get(index).getText(),
+                         JdiTestingIndexPageData.HEADER_MENU_ITEMS_TEXT.get(index)
+                     );
+                 });
```

Comment:
Usage of loops (and loop like constructions) usually is not a good idea
You have the assertJ in pom.xml, so I suggest to use its methods like `org.assertj.core.api.AbstractIterableAssert#allMatch()` and methods for collections equality  check 

\+ same for other places with loops and forEach with assertions inside

You can chat in private for discussion